srv/coordinator/coordinator.py

```diff
@@ -277,11 +278,18 @@ def save(self, job):
 
 
 @click.command()
-@click.option('-platform', required=True, type=str)
-@click.option('-database', required=True, type=str)
-@click.option('-messenger', required=True, type=str)
+@click.option('-version', count=True)
+@click.option('-platform', default='K8S', type=str)
+@click.option('-database', default='POSTGRESQL', type=str)
+@click.option('-messenger', default='POSTGRESQL', type=str)
```

Comment:
Two dashes for launch parameters are better, but it will lead to changes in the sh script and the helm chart.

Reply:
Let us leave them as it is for now because the scope of the change is big.  Later I will change it to double dashes.